sql/grant_user_to_database.sql.in

```diff
@@ -0,0 +1,3 @@
+CREATE USER '{RPCM_DB_USER}'@'{RPCM_LOCAL_IP}' IDENTIFIED BY '{RPCM_DB_PASS}';
+GRANT ALL ON {RPCM_DB_NAME}.* TO '{RPCM_DB_USER}'@'{RPCM_LOCAL_IP}';
```

Comment:
I think we should grant the user only needed permissions, don't you think so @celestian?

Reply:
Addressed.